webstudio-is/webstudio
# Figma tokens sync

## Description

We were unable to create and sync new tokens in Tokens Studio for Figma. I encountered this problem 3 weeks ago when trying to add color tokens for link text. Taylor also confirmed experiencing this issue. Today, I tried to reproduce the issue, but to my surprise, Tokens Studio for Figma worked as expected. 

If this pull request works, then our Tokens Studio is up to date and in sync with the Styles in the Figma Library 🎉

## Steps for reproduction

1. Open our Figma library file: https://www.figma.com/file/sfCE7iLS0k25qCxiifQNLE/%F0%9F%93%9A-Webstudio-Library?type=design&node-id=0%3A1&mode=design&t=bJaj3aaWfiGBa8Dg-1
2. Open the Tokens Studio for Figma Plugin, and make sure the correct Sync Provider is active—see screenshot below. (if not installed, then you have to get the plugin, and get it synced with our Github—I remember we have a document somewhere for this, but I can't find it)
3. Test if you can do these things:
  - Create a new style in Figma, then import it into the Tokens Plugin `Styles & Variables > Import Styles`. The new style should show up as a new design token in the plugin.
  - Edit this new design token and `Apply to Document`
  - `Push to Github`
  
  If these work, then it's clear that the Tokens Plugin is working and the issue we experience before is no longer a problem 

![image](https://github.com/webstudio-is/webstudio/assets/122043657/7283b872-0af3-46d7-8d05-067e0832ab92)


## Code Review

  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review


## Before merging
